### PR TITLE
redirected 404 request to home page and the home page to the travel mate website.

### LINKED ISSUE #41: [API Request] Add home route and 404 routes
Currently we don't have a 404 route or home route.
+ For 404 route, we should be **permanently** redirecting the user to home route.
+ For home route (`/`), we redirect the user to [Travel mate website](http://project-travel-mate.github.io/Travel-Mate)

### PR DESCRIPTION
**Pull request by arunikayadav42**
redirected 404 request to home page and the home page to the travel mate website.Added a views.py file to the api app and made changes in the urls.py file of the nomad app.

Fixes #41 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `flake8`
- [x] `python manage.py test`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: api/modules/currency/constants.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""
```
Comment (prabhakar267) [inline api/modules/currency/constants.py]: Remove unwanted files

**Review by prabhakar267 — CHANGES_REQUESTED** [review]

**Review by prabhakar267 — CHANGES_REQUESTED** [review]
Please test your code locally, it gives me an error `localhost redirected you too many times.`

File: api/views.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import redirect
+
+
+def view_404(request):
+    return redirect('/')
+
+
+def view_home(request):
+    return redirect('http://project-travel-mate.github.io/Travel-Mate/')
```
Comment (prabhakar267) [inline api/views.py]: Move this URL string to a new file `nomad/constants.py` and name the variable to `URL_NOT_FOUND_REDIRECT`

**Review by prabhakar267 — CHANGES_REQUESTED** [review]
404 is not working

**Comment by arunikayadav42** [comment]
exactly where is the fault now...can you please tell?

**Review by prabhakar267 — APPROVED** [review]

**Pull request merged**